### PR TITLE
sshd: first draft

### PR DESCRIPTION
**Pull request by JonasCir**

### CODE REVIEW

File: ansible/.gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 inventory/inventory.yml
 ansible.cfg
+roles/ssh-initial/vars/main.exmaple.yml
```
Comment (TheJKM) [inline ansible/.gitignore:3]: Why the example? Wouldn't be better to exclude `main.yml` with personal data?

**Review by TheJKM — CHANGES_REQUESTED** [review]
See my changes in #11

File: ansible/.gitignore (hunk 1 above)
Comment (JonasCir) [reply ansible/.gitignore:3]: the problem is that the format is not clear if you leave the file out

Comment (JonasCir) [reply ansible/.gitignore:3]: uh sorry you are correct

**Review by TheJKM — APPROVED** [review]

**Pull request merged**